src/amo/components/Category/index.js

```diff
@@ -251,7 +253,11 @@ export class CategoryBase extends React.Component {
     const { html } = this.contentForType(addonType);
 
     return (
-      <div className="Category">
+      <div
```

Comment:
I've been trying to figure out where this is used in the site to test it, but I cannot find it anywhere. Where is this actually used?

Reply:
I am doing this in a few places since I need to override the grid style for the themes. For example, the Category--theme styles look like this:

```css
.Category--theme {
  .Card-contents {
    .AddonsCard-list {
      grid-template-columns: repeat(3, 33%);
    }
  }
}
```

Reply:
should this be tested?

Reply:
I added a test for this..